pandas-stubs/core/frame.pyi

```diff
@@ -326,23 +327,39 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def to_html(
         self,
-        buf: FilePathOrBuffer | None,
-        columns: Sequence[_str] | None = ...,
-        col_space: int | list[int] | dict[_str | int, int] | None = ...,
+        buf: FilePath | WriteBuffer[str],
+        columns: Sequence[HashableT] | None = ...,
```

Comment:
This might be too broad (as was the previous version).  Will match a single string.  Use `list[HashableT] | tuple[HashableT,...]`

Reply:
Fixed.